src/StimulusBundle/README.md

```diff
@@ -1,12 +1,9 @@
 # StimulusBundle: Symfony integration with Stimulus!
 
-**EXPERIMENTAL** This bundle is currently experimental. It is possible that
-backwards-compatibility breaks could happen between minor versions.
-
 This bundle adds integration between Symfony, Stimulus and Symfony UX:
 
 -   A) Twig `stimulus_*` functions & filters to add Stimulus controllers, actions & targets in your templates;
--   B) Integration with Symfony UX & AssetMapper;
+-   B) Integration with Symfony UX & AssetMapper (this integration is **experimental**)
```

Comment:
It could be worth making `experimental` point to https://symfony.com/doc/current/contributing/code/experimental.html